multichain-testing/scripts/make-bank-asset-info.ts

```diff
@@ -3,7 +3,7 @@
 
 import '@endo/init';
 import starshipChainInfo from '../starship-chain-info.js';
-import { makeAssetInfo } from '../tools/asset-info.ts';
```

Comment:
This is a code change, not just lint change. I suspect it will break this test. I've added "force:integration" to check